src/Valit/ValitRuleStringExtensions.cs

```diff
@@ -19,5 +19,8 @@ public static class ValitRuleStringExtensions
 
         public static IValitRule<TObject, string> Email<TObject>(this IValitRule<TObject, string> rule) where TObject : class
             => rule.Matches(@"\A(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?)\Z");
+
+        public static IValitRule<TObject, string> Required<TObject>(this IValitRule<TObject, string> rule) where TObject : class
+            => rule.Satisfies(p => !string.IsNullOrEmpty(p));
```

Comment:
Should we also check on whitespace here?

Reply:
Every string extension has just null or empty. I'd stick with that also here